sdk/tests/deqp/functional/gles3/es3fNegativeShaderApiTests.js

```diff
@@ -435,38 +435,6 @@ goog.scope(function() {
             }
         ));
 
-        testGroup.addChild(new es3fApiCase.ApiCaseCallback('uniformf_invalid_location', 'Invalid glUniform{1234}f() usage', gl,
-            function() {
-                /** @type {gluShaderProgram.ShaderProgram} */ var program = new gluShaderProgram.ShaderProgram(gl, gluShaderProgram.makeVtxFragSources(uniformTestVertSource, uniformTestFragSource));
-
-                gl.useProgram(program.getProgram());
-                this.expectError(gl.NO_ERROR);
-
-                bufferedLogToConsole('gl.INVALID_OPERATION is generated if location is an invalid uniform location for the current program object and location is not equal to -1.');
-                gl.useProgram(program.getProgram());
-                gl.uniform1f(null, 0.0);
```

Comment:
native deqp tests use -2 (and -1) as the uniform location for these tests, see https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/gles3/functional/es3fNegativeShaderApiTests.cpp#809. Seems that we can not wrap -2 / -1 into WebGLUniformLocation variable. 

null is thought as 0, so the tests will not generate INVALID_OPERATION.